frontend/packages/dev-console/integration-tests/features/monitoring/monitoringa-Page-Smoke-Tests-ODC-4116.feature

```diff
@@ -0,0 +1,96 @@
+Feature: Monitoring Page
+    As a user, I should be able to perform actions related to monitoring a project and deployments inside it
+
+
+Background: 
+    Given user is at Developer perspective
+
+
+@smoke, @regression
+Scenario: Monitoring Page: MH-03-TC02
+    Given user is at Developer perspective
+    When user opens Monitoring 
+    Then user will see Dashboard tab
+    And user will see Metrics tab
+    And user will see Alerts tab
+    And user will see Events tab
```

Comment:
IMO, single line is sufficient to mention all the tab names instead of multiple lines

Reply:
Updated